proto/cork/v1/cork.proto

```diff
@@ -5,14 +5,24 @@ import "cosmos_proto/cosmos.proto";
 
 option go_package = "github.com/peggyjv/sommelier/v3/x/cork/types";
 
-// MsgSubmitCorkRequest - sdk.Msg for submitting calls to Ethereum through the gravity bridge contract
 message Cork {
   // call body containing the ABI encoded bytes to send to the contract
   bytes encoded_contract_call = 1;
   // address of the contract to send the call
   string target_contract_address = 2;
 }
 
+message ValidatorCork {
+  Cork cork = 1;
+  string validator = 2;
+}
+
+message ScheduledCork {
```

Comment:
Nit: should we make this a `ValidatorCork` and a `block_height`?

Reply:
Does this really help? The only place we use it is in queries, it would be one more thing for the end user to unwrap

Reply:
Eh, it's not necessary. Just repeating the same patterns of fields in many places means if you have to change it for some reason that has to be duplicated. Probably unlikely to be an issue here, we can leave as is.